packages/core/src/Calendar/CalendarCellTrigger.vue

```diff
@@ -85,6 +85,8 @@ const isSelectedDate = computed(() => rootContext.isDateSelected(props.day))
 
 const SELECTOR
   = '[data-reka-calendar-cell-trigger]:not([data-disabled]):not([data-outside-view]):not([data-outside-visible-view])'
+const NOT_OUTSIDE_VIEW_SELECTOR
```

Comment:
@hartbit I think you can replace `SELECTOR` with this altogether

Reply:
Thanks for the suggestion. I made the change, and indeed, it doesn't break any tests.